<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ورود به سامانه</string>
  </property>
  <widget class="QGraphicsView" name="graphicsViewBackground">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>411</width>
     <height>451</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="interactive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>201</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    background-color: transparent; 
    border: 2px solid white; 
    border-radius: 8px;
    font-size: 16px;
}
QLabel {
    color: white; 
    font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>فرم ورود</string>
   </property>
   <widget class="QLabel" name="lineEditUsername">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>نام کاربری:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>44</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>رمز عبور:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>26</y>
      <width>118</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonConfirm">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>110</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>تأیید</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCancel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>110</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>لغو</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelLogo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/logos/Images/logo.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompanyInfo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    background-color: transparent; 
    border: 2px solid white; 
    border-radius: 8px;
    font-size: 16px;
}
QLabel {
    color: white; 
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>www.comnersa.ir</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>7</number>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelSoftwareName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>9</width>
     <height>9</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>9</width>
     <height>9</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>36</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: orange;</string>
   </property>
   <property name="text">
    <string>رسا</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>7</number>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelSoftwareDescription">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>9</width>
     <height>9</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>9</width>
     <height>9</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffaa7f;</string>
   </property>
   <property name="text">
    <string>نرم افزار صورت جلسه نویسی</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>7</number>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelLocation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>9</width>
     <height>9</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>9</width>
     <height>9</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    background-color: transparent; 
    border: 2px solid white; 
    border-radius: 8px;
    font-size: 16px;
}
QLabel {
    color: white; 
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>آدرس: پیچ شمیران - بن بست ریحانی پلاک 1 - طبقه دوم شرقی تلفن تماس: 09199524011</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>7</number>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
